tests/unit_tests/gui/test_csv_export.py

```diff
@@ -72,3 +74,45 @@ def handle_finished_box():
                 f",{name},{gen_kw_data.iloc[i]['COEFFS:a']},{gen_kw_data.iloc[i]['COEFFS:b']},{gen_kw_data.iloc[i]['COEFFS:c']},{misfit_data.iloc[i]['MISFIT:POLY_OBS']},{misfit_data.iloc[i]['MISFIT:TOTAL']}"
                 in file_content
             )
+
+
+@pytest.mark.parametrize("ensemble_select", ["default_0", "*"])
+def test_csv_export(esmda_has_run, qtbot, ensemble_select):
+    gui = esmda_has_run
+
+    file_name = export_data(gui, qtbot, ensemble_select)
+    verify_exported_content(file_name, gui, ensemble_select)
+
+
+def run_experiment_and_export(gui, qtbot):
+    experiment_panel = get_child(gui, ExperimentPanel)
+    simulation_mode_combo = get_child(experiment_panel, QComboBox)
+    simulation_mode_combo.setCurrentText(EnsembleExperiment.name())
+    ensemble_experiment_panel = get_child(experiment_panel, EnsembleExperimentPanel)
+    ensemble_experiment_panel._ensemble_name_field.setText("iter-0")
+
+    # Avoids run path dialog
+    with contextlib.suppress(FileNotFoundError):
+        shutil.rmtree("poly_out")
+
+    run_experiment = get_child(experiment_panel, QWidget, name="run_experiment")
+    qtbot.mouseClick(run_experiment, Qt.LeftButton)
+
+    run_dialog = wait_for_child(gui, qtbot, RunDialog)
+    qtbot.waitUntil(run_dialog.done_button.isVisible, timeout=100000)
+    qtbot.waitUntil(lambda: run_dialog._tab_widget.currentWidget() is not None)
+    qtbot.mouseClick(run_dialog.done_button, Qt.LeftButton)
+
+
+def test_that_export_tool_does_not_produce_duplicate_data(
+    ensemble_experiment_has_run_no_failure, qtbot
+):
+    gui = ensemble_experiment_has_run_no_failure
+
+    run_experiment_and_export(gui, qtbot)
+
+    file_name = export_data(gui, qtbot, "*")
+
+    df = pd.read_csv(file_name)
+    # Make sure data is not duplicated.
+    assert df.iloc[0]["COEFFS:a"] != df.iloc[20]["COEFFS:a"]
```

Comment:
This requires some explanation. I guess it should be 10 rows of 1 run experiment and 10 rows with another run and this will make sure that the values are not duplicate? But then we should firstly assert that both runs have the same experiment name?

Reply:
one more thingy: `assert exp1_name==exp2_name`

Reply:
Good idea, I've pushed a fix.